gitextensions/gitextensions
# Remove splash screen (#1871)

Fixes #1871. In this branch splash screen removed at all

What did I do to test the code and ensure quality:
 - run GitExtensions
 - make sure that splash screen does not appear

Has been tested on (remove any that don't apply):
 - GIT 2.14 and above
 - Windows 10 and above

## SynchronizationContext workaround
When I removed splash screen SynchronizationContext.Current became null. After an hour of investigation I found that it creates on first UI form creation. So I decide to create temporary empty form and get SynchronizationContext.Current. And it works.
It is some kind of workaround but there already was a workaround as commented

> // Store here SynchronizationContext.Current, because later sometimes it can be null
> //see http://stackoverflow.com/questions/11621372/synchronizationcontext-current-is-null-in-continuation-on-the-main-ui-thread
